.ci/targets/ios_platform_tests.yaml

```diff
@@ -21,7 +21,7 @@ tasks:
     args: ["xcode-analyze", "--ios", "--ios-min-version=13.0"]
   - name: native test
     script: script/tool_runner.sh
-    args: ["native-test", "--ios", "--ios-destination", "platform=iOS Simulator,name=iPhone 14,OS=latest"]
+    args: ["native-test", "--ios", "--ios-destination", "platform=iOS Simulator,name=Flutter-iPhone,OS=16.4"]
```

Comment:
Due to the changes in the way runtimes are handled in CI for macOS 13, we need to be more specific when targeting a simulator device.